src/FluentValidation.Tests.AspNetCore/StartupWithMvcValidationDisabled.cs

```diff
@@ -31,8 +33,15 @@ public void ConfigureServices(IServiceCollection services)
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
         public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerFactory loggerFactory)
         {
-            app.UseMvc(routes =>
-            {
+	        CultureInfo cultureInfo = new CultureInfo("en-US");
+
+	        app.UseRequestLocalization(options => {
+		        options.DefaultRequestCulture = new RequestCulture(cultureInfo);
+		        options.SupportedCultures = new[] { cultureInfo };
+		        options.SupportedUICultures = new[] { cultureInfo };
+	        });
+
+			app.UseMvc(routes => {

```

Comment:
In my computer, this line does not seen like this. I try to fix this

Reply:
It's probably an issue tabs vs spaces. Turn on whitespace in VS and it should show you where the difference is (Edit -> Advanced -> View Whitespace)